templates/scenarios/common/office-addin/teamsapp.yml

```diff
@@ -16,14 +16,12 @@ provision:
          parameters: ./infra/azure.parameters.json # Relative path to this file. Placeholders will be replaced with corresponding environment variable before ARM deployment.
          deploymentName: Create-resources-for-tab # Required when deploy ARM template
       bicepCliVersion: v0.9.1 # Teams Toolkit will download this bicep CLI version from github for you, will use bicep CLI in PATH if you remove this config.
-    # Output: every bicep output will be persisted in current environment's .env file with certain naming conversion. Refer https://aka.ms/teamsfx-actions/arm-deploy for more details on the naming conversion rule.
 
   - uses: azureStorage/enableStaticWebsite
     with:
       storageResourceId: ${{ADDIN_AZURE_STORAGE_RESOURCE_ID}}
       indexPage: index.html
       errorPage: error.html
-    # Output: N/A
```

Comment:
@Siglud I also removed the output comment for the azure storage action, please help review